native/src/routes/settings/components/Settings.js

```diff
@@ -134,7 +135,8 @@ export default class Settings extends React.Component<PropsType, StateType> {
       return null
     }
 
-    return <SectionList
+    return <LayoutContainer>
```

Comment:
Adding a container here means that Settings becomes a Container. This should be moved to the upper level in the component tree.

Reply:
What do you mean by `should be moved to upper level in the component tree`?

Reply:
Settings is in the components directory. By using LayoutContainer the component is connected to the redux state. So eithger move Settings to container directory or fixis by not using layoutcontainer there.

This already was a problem before so I'm fine with not fixing it.

Reply:
Actually is LayoutContainer just there because of `withTheme`, no redux.